modules/ROOT/pages/cloud-deployments/neo4j-cluster-cloud.adoc

```diff
@@ -0,0 +1,109 @@
+:description: Tha page describes how to manage the Neo4j cluster on AWS.
+:page-role: enterprise-edition
+
+[[neo4j-cluster-cloud-deployments]]
+= Neo4j cluster on self-managed cloud deployments
+
+Before diving into the topic, it is important to understand basics about Neo4j's clustering.
+
+Neo4j cluster consists of a homogenous pool of servers that collectively run a number of databases.
+The servers can operate in two different database-hosting modes: _primary_ and _secondary_.
+A server can simultaneously act as a primary host for one or more databases and as a secondary host for other databases.
+
+For more details on operational and application aspects of Neo4j's clustering, refer to the xref::clustering/index.adoc[Clustering in Neo4j].
+
+For information on how to manage databases and servers in a cluster, see respectively xref::clustering/databases.adoc[] and xref::clustering/servers.adoc[].
+
+
+== Neo4j cluster on AWS
+
+Neo4j does not provide Amazon Machine Images (AMIs) with a pre-installed version of the product.
+The Neo4j AWS Marketplace listings (and listings on GitHub) use CloudFormation templates that deploy and configure Neo4j dynamically with a shell script.
+
+
+// === Neo4j cluster and auto-scaling groups on AWS
+
+
+=== Removing a (secondary constrained) server from the cluster
```

Comment:
Are the following instructions applicable to the secondaries only? What to do if you need to remove a primary constrained server?

Reply:
No, this applies to any server.